content/Sessions/Sessions.facts.md

```diff
@@ -0,0 +1,42 @@
+### Facts for Sessions lesson:
+
+A session is a way to store information state and history of user activities in a certain application using a unique session ID. 
+
+The `session_start()` function creates new or resumes a certain session.
+
+The `session_destroy()` function destroys the entire session variables in PHP.
+
+The `unset()` function is used to destroy a specific session variable.
+
+The `$_SESSION` superglobal variable is used to store and access session data.
+
+The `session_id()` function sets and accesses a session ID. By default, PHP sessions generate a session ID randomly.
+
+The `$_POST` superglobal variable is used to collect form data submitted through the HTTP POST method.
```

Comment:
Haven't we introduce this already in the previous PHP course?

Reply:
If not, can we separate this as a request method together with $_GET and $_REQUEST?